<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CDAparaArreraLib</class>
 <widget class="QDialog" name="CDAparaArreraLib">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>700</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>700</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>700</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Arrera : Librairy parametre</string>
  </property>
  <widget class="QFrame" name="Fapp">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>221</width>
     <height>371</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>111</width>
      <height>371</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="leftMargin">
      <number>5</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="IDC_GESTAPP1">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>40</height>
        </size>
       </property>
       <property name="text">
        <string>Application 1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="IDC_GESTAPP2">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>40</height>
        </size>
       </property>
       <property name="text">
        <string>Application 2</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="IDC_GESTAPP3">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>40</height>
        </size>
       </property>
       <property name="text">
        <string>Application 3</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="IDC_GESTAPP4">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>40</height>
        </size>
       </property>
       <property name="text">
        <string>Application 4</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="IDC_GESTAPP5">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>40</height>
        </size>
       </property>
       <property name="text">
        <string>Application 5</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="IDC_GESTAPP6">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>40</height>
        </size>
       </property>
       <property name="text">
        <string>Application 6</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>0</y>
      <width>111</width>
      <height>371</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <property name="leftMargin">
      <number>5</number>
     </property>
     <item>
      <widget class="QPushButton" name="IDC_GESTAPP7">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>40</height>
        </size>
       </property>
       <property name="text">
        <string>Application 7</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="IDC_GESTAPP8">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>40</height>
        </size>
       </property>
       <property name="text">
        <string>Application 8</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="IDC_GESTAPP9">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>40</height>
        </size>
       </property>
       <property name="text">
        <string>Application 9</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="IDC_GESTAPP10">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>40</height>
        </size>
       </property>
       <property name="text">
        <string>Application 10</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="IDC_GESTAPP11">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>40</height>
        </size>
       </property>
       <property name="text">
        <string>Application 11</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="IDC_GESTAPP12">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>40</height>
        </size>
       </property>
       <property name="text">
        <string>Application 12</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QFrame" name="FArreraApp">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>380</y>
     <width>221</width>
     <height>321</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>221</width>
      <height>321</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <property name="leftMargin">
      <number>40</number>
     </property>
     <item>
      <widget class="QPushButton" name="IDC_GESTARRERARECHERCHE">
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>120</width>
         <height>40</height>
        </size>
       </property>
       <property name="text">
        <string>Arrera Recherche</string>
       </property>
       <property name="icon">
        <iconset resource="ArreraTheme.qrc">
         <normaloff>:/iconBTN/ArreraRecherche.png</normaloff>:/iconBTN/ArreraRecherche.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="IDC_GESTARRERAINFO">
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>120</width>
         <height>40</height>
        </size>
       </property>
       <property name="text">
        <string>Arrera Info</string>
       </property>
       <property name="icon">
        <iconset resource="ArreraTheme.qrc">
         <normaloff>:/iconBTN/ArreraInfo.png</normaloff>:/iconBTN/ArreraInfo.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="IDC_GESTARRERAVIDEO">
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>120</width>
         <height>40</height>
        </size>
       </property>
       <property name="text">
        <string>Arrera Video</string>
       </property>
       <property name="icon">
        <iconset resource="ArreraTheme.qrc">
         <normaloff>:/iconBTN/ArreraVideoDownload.png</normaloff>:/iconBTN/ArreraVideoDownload.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="IDC_GESTARRERADOC">
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>120</width>
         <height>40</height>
        </size>
       </property>
       <property name="text">
        <string>Arrera Doc</string>
       </property>
       <property name="icon">
        <iconset resource="ArreraTheme.qrc">
         <normaloff>:/iconBTN/ArreraDoc.png</normaloff>:/iconBTN/ArreraDoc.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QFrame" name="FGESTAPP">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>10</y>
     <width>361</width>
     <height>681</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="IDC_LTITRE">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>341</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Gestion Application</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="IDC_LAPPSELECTIONNER">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>331</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="IDC_LNAMEAPP">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Nom : </string>
    </property>
   </widget>
   <widget class="QTextEdit" name="IDC_TEXTAPPNAME">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>170</y>
      <width>281</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="IDC_VALIDER">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>280</y>
      <width>341</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Valider</string>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="FGESTTASKBAR">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>591</width>
     <height>691</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="IDC_LTASKBAR">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>541</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Gestion taskbar</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QSpinBox" name="IDC_CHOXBTN">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>220</y>
      <width>51</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>12</number>
    </property>
   </widget>
   <widget class="QPushButton" name="IDC_VALIDERTASKBAR">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>630</y>
      <width>100</width>
      <height>35</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="ArreraTheme.qrc">
      <normaloff>:/iconBTN/valider.png</normaloff>:/iconBTN/valider.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>100</width>
      <height>35</height>
     </size>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>50</y>
      <width>160</width>
      <height>631</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <property name="leftMargin">
      <number>40</number>
     </property>
     <item>
      <widget class="QPushButton" name="IDC_TASK1">
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>70</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>70</width>
         <height>70</height>
        </size>
       </property>
       <property name="text">
        <string>Raccourcie 
1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="IDC_TASK2">
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>70</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>70</width>
         <height>70</height>
        </size>
       </property>
       <property name="text">
        <string>Raccourcie 
2</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="IDC_TASK3">
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>70</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>70</width>
         <height>70</height>
        </size>
       </property>
       <property name="text">
        <string>Raccourcie 
3</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="IDC_TASK4">
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>70</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>70</width>
         <height>70</height>
        </size>
       </property>
       <property name="text">
        <string>Raccourcie 
4</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="IDC_TASK5">
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>70</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>70</width>
         <height>70</height>
        </size>
       </property>
       <property name="text">
        <string>Raccourcie 
5</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="IDC_LBARAPP">
    <property name="geometry">
     <rect>
      <x>178</x>
      <y>229</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Numero de l'application :</string>
    </property>
   </widget>
   <widget class="QLabel" name="IDC_LRACOURCISELECT">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>110</y>
      <width>281</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <zorder>FArreraApp</zorder>
  <zorder>Fapp</zorder>
  <zorder>FGESTAPP</zorder>
  <zorder>FGESTTASKBAR</zorder>
 </widget>
 <resources>
  <include location="ArreraTheme.qrc"/>
 </resources>
 <connections/>
</ui>
